### DISCUSSION TITLE
D3 Support for skipping holidays and  weekends in a timescale

### DISCUSSION THREAD

**Discussion by Serjohn27** [d3/d3#3480] [Q&A] [answered]
Hi, 
Is there any option or potential hack in d3 to support a discontinuous scale? I was using library for stock price charting, since market is closed at weekends and certain holidays, these data points create gaps in chart ( which is maybe normal behavior for other use cases ), is there a way to somehow get rid of those gaps ?

**Comment by gordonwoodhull** [comment, marked as answer]
I think the usual practice is to use an ordinal scale for this.

**Comment by mbostock** [comment]
See this example: https://observablehq.com/@d3/candlestick-chart
